code-conventions.md

```diff
@@ -100,14 +100,39 @@ If we need to remove or alter contributed code due to a licensing issue we will
   - When omitting brackets, still indent as if the code had brackets
   - Avoid omitting brackets if it produces hard-to-read code
 * Annotations for methods and classes are placed in lines before their declarations, one per line
+* All syntaxes should have proper Skript documentation annotations. (Located in package ...skript.doc.*)
+  - When declaring Skript documentation annotations with multiple lines, seperate them into a String list aswell as placing each on a new line.
+  - When it comes to **simple short examples**, it's a better format to include indentations in the actual class for simplier reading. (See the first example in the Examples annotation).
+  - We use the format character `\t` to represent the tab character in our documentation.
+  - Seperate multiple examples with an empty string. (See Examples annotation below).
+  - Use inline condition statements over multiple if statements when possible to avoid arrow code format. (See the second example in the Examples annotation compared to the first example).
+```java
+@Description({
+	"This is a multiple line documentation example",
+	"for the Skript documentation annotations."
+})
+```
+```java
+@Examples({
+	"on damage of a player:",
+		"\tif victim is wearing a diamond helmet:",
+			"\t\tmessage \"String example\" to victim",
+	"",
+	"on inventory click:",
```

Comment:
I think indentations shouldn't be used only when there are 3+ indentations don't you think?

Reply:
I think 3 should be the limit yes, but it's also explained in another bullet point to use inline conditions when possible, so it shouldn't get to a length of more than 3.